FirebaseInAppMessaging/Swift/Source/CustomInAppMessageDisplayViewModifier.swift

```diff
@@ -11,12 +11,13 @@ struct ImageOnlyInAppMessageDisplayViewModifier<DisplayMessage: View>: ViewModif
     return content.overlay(overlayView())
   }
 
+  @ViewBuilder
   func overlayView() -> some View {
     if let imageOnlyMessage = delegateBridge.inAppMessageData?.0 as? InAppMessagingImageOnlyDisplay,
       let delegate = delegateBridge.inAppMessageData?.1 {
-      return AnyView(closure(imageOnlyMessage, delegate))
+      closure(imageOnlyMessage, delegate)
     }
-    return AnyView(EmptyView())
+    EmptyView()
```

Comment:
Nit: The `EmptyView()` should be in an else for consistency - it doesn't need to be there when the `if let` succeeds. it doesn't have any effect on the UI but from a code structure at least.

Here and below, as well as the other files in these 2 commits.

Reply:
Resolved in dd358fc.